src/register/mod.rs

```diff
@@ -30,19 +30,23 @@ pub mod utval;
 pub mod fcsr;
 
 // User Counter/Timers
-// TODO: cycle[h], instret[h]
-pub mod time;
-#[rustfmt::skip] // long macro use
+pub mod cycle;
 mod hpmcounterx;
+pub mod instret;
+pub mod time;
+
 pub use self::hpmcounterx::*;
```

Comment:
```suggestion
mod hpmcounterx;
pub use self::hpmcounterx::*;
pub mod instret;
pub mod time;
```

It's strage to separate `mod` and `pub use` for the same module by several lines.

Reply:
I think cargo fmt did that, I don't remember doing that.  Oh well, I fixed it :)